Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-syntax-highlight bundle size difference from 3.0.0 to 5fb0e58681e68e099ac07a66189c725742de3c39
js: 0.67kb increase (0.27kb/gzip)
An insignificant difference was also found for: css (master), css (internal)

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-syntax-highlight bundle size difference from 3.0.0 to 056f0ef58e69aac7df4883481c9de49583108508
js: 0.67kb increase (0.27kb/gzip)
An insignificant difference was also found for: css (master), css (internal)

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
